csharp/ql/src/Likely Bugs/UncheckedCastInEquals.ql

```diff
@@ -35,4 +35,4 @@ where
   access = cast.getAChild() and
   access.getTarget().getDeclaringElement() = access.getEnclosingCallable() and
   nodeBeforeParameterAccess(access.getAControlFlowNode())
-select cast, "Missing type-check before casting parameter to 'Equals'."
+select cast, "Equals() method does not seem to check argument type."
```

Comment:
Shouldn't `@precision high` mean high confidence in the result? If so, then we could change `does not seem to check` to `does not check`.